config.go

```diff
@@ -164,6 +164,11 @@ func (p PeerInfo) HashKey() string {
 
 type UpdateFunc func([]PeerInfo)
 
+type LogConfig struct {
```

Comment:
It doesn't look like we need a struct to define these fields.  Can they just be variables in `SetupDaemonConfig()`?